docs/Coding-Guidelines/HybridCRT.md

```diff
@@ -131,3 +126,47 @@ all projects in the repository via this statement:
 ```
 
 This applies to all projects in the repository (product, test, ...).
+
+
+## C++/CX Special Considerations
+
+Some feeder repos make extensive use of CX in test code.  This logic is often linked with production logic, which complicates the adoption of Hybrid CRT.  There are a few build configuration changes to resolve the resulting conflicts, described here.  
```

Comment:
Eh. Remove mention of feeders. The docs are written for Foundation but can be applied to others (e.g. DWriteCore).